app/views/design-system/rsd-design-system/design-patterns/footer/index.html

```diff
@@ -0,0 +1,176 @@
+          All content is available under the
+          &lt;a class=&quot;govuk-footer__link&quot; href=&quot;https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/&quot; rel=&quot;license&quot;&gt;Open
+            Government Licence v3.0&lt;/a&gt;, except where otherwise stated
+        &lt;/span&gt;
+      &lt;/div&gt;
+      &lt;div class=&quot;govuk-footer__meta-item&quot;&gt;
+        &lt;a class=&quot;govuk-footer__link govuk-footer__copyright-logo&quot; href=&quot;https://www.nationalarchives.gov.uk/information-management/re-using-public-sector-information/uk-government-licensing-framework/crown-copyright/&quot;&gt;
+          &copy; Crown copyright
+        &lt;/a&gt;
+      &lt;/div&gt;
+    &lt;/div&gt;
+  &lt;/div&gt;
+&lt;/footer&gt;
+</code>' %}
+
+{% block beforeContent %}{% endblock %}
+
+{% block content %}
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+
+    {{ sample.code('/design-system/rsd-design-system/design-patterns/footer/example.html', 'Footer', 'footer',
+    htmlCode) }}
+    <br>
+
+    <h2>When to use this component</h2>
+    <p>Use this pattern to create a product footer that is consistent with other RSD products.</p>
+    <figure class="govuk-!-margin-bottom-9" style="margin: 0;">
+      <img src="/assets/images/rsd-pattern-library-footer-screencapture-primary.png"
+        alt="A footer with Regional Services Division links." width="100%">
+      <figcaption>Screenshot of the RSD footer design pattern</figcaption>
+    </figure>
+
+    <h2>When not to use this component</h2>
+    <p>On</p>
+    <ul>
+      <li>external-facing sites, like <a href="https://apply-convert-academy.service.gov.uk/">Apply to become
+          an academy (A2B)</a> This site is for external users and the Microsoft links in this pattern
+        won't work for them</li>
+      <li>SharePoint guidance pages, like <a
+          href="https://educationgovuk.sharepoint.com/sites/lvewp00299/SitePages/Prepare-Conversions-and-Transfers.aspx">Prepare
+          conversions and transfers - SharePoint</a></li>
+    </ul>
+
+    <h2>Links</h2>
+    <p>The RSD footer includes several links:</p>
+    <h3>Email Service Support for help with using this system</h3>
+    <p>This is a mail-to link with the subject header:</p>
+    <code>[Product name]: support query</code>
+    <p>For example, Prepare conversions and transfers: support query</p>
+    <p> It has this text underneath the link: "We aim to respond within 5 working days, or one working day for
+      more urgent queries." </p>
+    <h3>How to use this system (opens in a new tab)</h3>
+    <p> This links to the section of a product's SharePoint page with guidance on how to use the product. For
+      example, <a
+        href="https://educationgovuk.sharepoint.com/sites/lvewp00299/SitePages/Prepare-Conversions-and-Transfers.aspx#guidance-for-using-prepare-conversions-and-transfers">Guidance
```

Comment:
In the "Our approach" page you open the intranet pages in a new tab, do the same here so it's consistent, and also use the hidden / visible text to say it's an intranet page being opened. (as it's only available to DfE staff).

Reply:
We updated the text for all "User needs database" links so that it contains the words "(opens in DfE Microsoft List)" now.